package.json

```diff
@@ -23,7 +23,7 @@
 	"scripts": {
 		"test": "npm run test:unit && npm run test:types",
 		"test:unit": "tap",
-		"test:types": "tsc --noEmit --strict src/index.d.ts",
+		"test:types": "tsc --noEmit --strict --target ES2018 --moduleResolution node --resolveJsonModule src/index.d.ts",
```

Comment:
Should the `--target` value here be updated to match the `ecmaVersion` in the ESLint config (`2020`)?

Reply:
I'm not clear why `--moduleResolution node` and `--resolveJsonModule` are needed here. Could you please explain?